testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -22,6 +22,10 @@
 from six.moves import cPickle
 
 import os
+try:
```

Comment:
This is annoying indeed. I just rediscovered `mda.lib.util.isstream` that would be better than `isinstance`. It looks at the capacity we need instead of the type. This is much better because it allows duck-typing where `isinstance` is much more rigid.

I check and it works with python 2. It should work with python 3, if it doesn't it is an other issue.